docs/application/tizen-studio/release-notes/4-0-release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Tizen Studio 4.0 Release Notes
+
+- Release Date: Oct 31, 2020
```

Comment:
I think @manishtr would also know, that the release date is changed again.
At here it is said that Tizen Studio 4.0 is planned to be published on Oct 29.
So, would you please update the release date to the actual date?

I hope this comment be the last update request...

Reply:
Thanks, i have done it